_docs/principles/accessibility.md

```diff
@@ -198,61 +303,71 @@ Helix minimum spacing standards require: 
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Provide easily identifiable feedback
+### Don't use tables for layout
 
-Provide feedback for interactions, such as confirming form submission, alerting the user when something goes wrong, or notifying the user of changes on the page. Ensure instructions are easy to identify and provide suggestions to help fix issues whenever errors are returned. Prominently present any important feedback that requires user action.
+Do not use tables to format content or for anything other than presenting tabular data. Screen readers use row and column headings to describe the content within each cell. If column or row headings aren't present, a screen reader may announce "Row one, Column Two..." which can be confusing to the user.
 
 {% endcolumn %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+</div>
 
-#### Failed text inputs
+</section>
 
-There are many acceptable ways to make text inputs visually accessible. If you're indicating a failed text input, make sure to include multiple visual indicators. For example, in addition to the error message, the border color turns red to draw the eye and doubles in thickness from 1px to 2px to provide another distinguishable visual cue that is not reliant on color.
+<section class="static-section"  markdown="1">
 
-It is important to reserve the prominent border at 2px for failed text input as an additional visual indicator for error an state.
+<div class="hxRow"  markdown="1">
 
-{% endcolumn %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+## Interactivity guidelines
```

Comment:
the spacing between headings and body content seems off...

Reply:
I see what you mean, but that spacing is correct.